dom/schema/v1/tollgate.sql

```diff
@@ -0,0 +1,9 @@
+create table if not exists tollgate
+(
+    id         text        not null primary key,
+    name       text        not null,
+    b_boxes    jsonb,
+    gate_line  jsonb,
```

Comment:
What exactly will be stored in `jsonb`? Do we really need this complex type?

Reply:
in b_boxes will be stored arbitrary array of bounding boxes, each box is 4 locations, and boxes_required attribute. or null.
in gate_line a set of coordinates representing line segment or null. 
jsonb type fits nicely for these use cases, because the exact data for storage not finalised yet.. 

not null value in either column 'triggers' corresponding algorithm for detecting tollgate crossing.